src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -66,7 +66,8 @@ public class PreferenceSettings : NotificationObject, IPreferences, IRenderPreci
         /// <summary>
         /// Indicates whether Google analytics reporting is approved or not.
         /// </summary>
-        public bool IsAnalyticsReportingApproved { get; set; }
+        [Obsolete("Property will be deprecated in Dynamo 3.0")]
+        public bool IsAnalyticsReportingApproved { get { return false; } set { } }
```

Comment:
I don't think this should be removed.

Reply:
I believe `IsUsageReportingApproved` which is already obsoleted is instrumentation which we want to remove.

Reply:
@mjkkirschner is on point here